### DISCUSSION TITLE
about Kernel Version for motr

### DISCUSSION THREAD

**Discussion by yanqingfu** [Seagate/cortx#433] [Q&A] [answered]
https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst#prerequisite

it mentioned 

> Different kernel versions that come from Centos7.7 or RHEL7.7 are supported.

here different kernel versions mean? 3.8 something or 3.10.0-1062.el7.x86_64 or 3.10.0-1062.el7.x86_32

**Comment by shailesh-vaidya** [comment, marked as answer]
I feel it will be always 64 bit with different Kernel versions like 3.10.0-1062.el7.x86_64 , 3.10.0-1127.el7.x86_64 etc.  
@madhavemuri  Can provide more details.

**Reply by huanghua78** [reply]
Yes, x86_64 arch.
